jax_unirep/evotuning.py

```diff
@@ -136,8 +160,29 @@ def length_batch_input_outputs(
     return xs, ys
 
 
+def get_batch_len(batched_seqs):
```

Comment:
Let's add type annotations here as well:

```suggestion
from typing import Iterable, Tuple, List
def get_batch_len(batched_seqs: Iterable[str]) -> Tuple[np.ndarray, List]:
```